scripts/dockerfiles/rsyslog_conf/etc/logrotate.d/rsyslog

```diff
@@ -1,14 +1,13 @@
 /var/log/syslog
 {
 	rotate 7
-	size 5M
+	maxsize=10M
+	minsize=9M
 	missingok
 	notifempty
 	delaycompress
 	compress
-	postrotate
-		reload rsyslog >/dev/null 2>&1 || true
-	endscript
+	copytruncate
```

Comment:
We should be able to `killall -HUP rsyslogd` instead of `copytruncate`, but both are documented to lose data so it doesn't matter.